.github/workflows/ci.yml

```diff
@@ -103,6 +103,10 @@ jobs:
       run: |
         go get -v -t -d ./...
         make install-tools
+    
+    - name: Automatic Docs Generation
+      run: |
+        make docs        
```

Comment:
On second thought what do you think if we add it below to the make doccheck and merge the two, so we have on doc check?